### DISCUSSION TITLE
Using EasyOC for state management ?

### DISCUSSION THREAD

**Discussion by giannik** [EasyOC/EasyOC#14] [Q&A]
Is there any suggestion for using easyOC for managing workflow states for content items ?
Example for an approval process define the specific states for an ApprovalRequest content item like :

- Submitted
- Pending Approval
- Need more info
- Rejected
- Approved.

Then using a task list to filter items based on state (eg all approval requests in state Pending Approval etc) 
and then changing the state to a new state ?
Thanks

**Comment by hyzx86** [comment]
Do not expose OC workflows directly to users; OC workflows are for developers, not users
You can create your own list, define your approval status arbitrarily, and then update it through the workflow
